helm_deploy/prison-offender-events/templates/_envs.tpl

```diff
@@ -88,76 +88,76 @@ env:
         name: dps-rds-instance-output
         key: rds_instance_endpoint
 
-  - name: SNS_AWS_ACCESS_KEY_ID
+  - name: HMPPS_SQS_TOPICS_PRISON_EVENT_TOPIC_ACCESS_KEY_ID
```

Comment:
Hmmm not sure this will work - think you might need `HMPPS_SQS_TOPICS_PRISONEVENTTOPIC_ACCESS_KEY_ID` - Spring tries its best to map env vars to properties but it's a really hard thing once you start thinking about camel case, kebab case etc.

Reply:
Will update